source/functions.lua

```diff
@@ -120,19 +120,38 @@ function functions.LoadGame()
 
     local savefile
     local contents
+	local size
+	local error = false
 
     savefile = savedir .. "/" .. "landers.dat"
-    contents, _ = nativefs.read( savefile)
-    garrLanders = bitser.loads(contents)
+
+	if love.filesystem.getInfo(savefile) ~= nil then

```

Comment:
This will result in error being true when the file has been found. Better just change to
```lua
if not love.filesystem.getInfo(savefile) then
```
 not double negate the value by acident.

Reply:
I was wrong. all good :heart: